assets/css/common/post-single.css

```diff
@@ -210,7 +210,7 @@
     display: block;
     margin: auto 0;
     padding: 10px;
-    color: rgba(255, 255, 255, .8);
+    color: var(--primary);
```

Comment:
Why is this changed? Does it have anything to do with the rest of the PR?

**Edit:** it appears to break the look of code blocks in light mode.

Reply:
Arg, I should not have used my `master` branch for the PR 🙈 I reverted this change there 👍 (b46ada5)

Reply:
Thanks! It might make sense to rebase on top of upstream's master, so there aren't two extra commits, but that's up to you and/or @adityatelange.